summingbird-core-test/src/test/scala/com/twitter/summingbird/memory/MemoryLaws.scala

```diff
@@ -87,7 +67,7 @@ object MemoryLaws extends Specification {
    * service functions.
    */
   def leftJoinLaw[T: Manifest: Arbitrary, K: Arbitrary, U: Arbitrary, JoinedU: Arbitrary, V: Monoid: Arbitrary: Equiv] = {
-    import MemoryArbitraries._
+    import com.twitter.summingbird.memory.MemoryArbitraries._
```

Comment:
The package location of these hasn't changed, so the imports shouldn't need to either?